CMakeLists.txt

```diff
@@ -24,6 +24,10 @@ set(dbusmenu_lxqt_lib_PATCH_VERSION 0)
 
 set(dbusmenu_lxqt_lib_VERSION ${dbusmenu_lxqt_lib_SOVERSION}.${dbusmenu_lxqt_lib_API_VERSION}.${dbusmenu_lxqt_lib_PATCH_VERSION})
 
+if(WITH_DOC)
+find_package(Doxygen REQUIRED)
```

Comment:
A 4-space indentation here?

Also, please set `WITH_DOC` to `OFF` near the start of this file. For now, it may be better to disable documentation instead of removing it altogether.